pkg/infra/game/stage/election/election.go

```diff
@@ -9,14 +9,24 @@ import (
 	"infra/game/state"
 )
 
-func HandleElection(state *state.State, agents map[commons.ID]agent.Agent, strategy decision.VotingStrategy, numberOfPreferences uint) (commons.ID, decision.Manifesto, uint) {
+func HandleElection(state *state.State, agents map[commons.ID]agent.Agent, strategy decision.VotingStrategy, numberOfPreferences uint) (
+	commons.ID, decision.Manifesto,
+) {
 	// Get manifestos from agents
 	agentManifestos := make(map[commons.ID]decision.Manifesto)
 
 	for id, a := range agents {
 		agentManifestos[id] = *a.SubmitManifesto(state.AgentState[id])
 	}
 
+	aliveAgentIDs := make([]commons.ID, 0)
+
+	for id, a := range state.AgentState {
+		if a.Hp > 0 {
+			aliveAgentIDs = append(aliveAgentIDs, id)
+		}
+	}
```

Comment:
```suggestion
	agentIDs := make([]commons.ID, len(agents))

	i := 0
	for k := range agents {
		agentIDs[i] = k
		i++
	}
```

I think that since we update the agentMaps upon death (i.e. deletion) - we needn't worry about performing this check.